packages/destination-actions/src/destinations/salesforce/lead/index.ts

```diff
@@ -0,0 +1,98 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const API_VERSION = 'v53.0'
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Lead',
+  description: 'Lead action',
+  fields: {
+    crud: {
```

Comment:
```suggestion
    operation: {
```

Wonder if this field name is more appropriate given:
1. CRUD isn't what we are doing since we may not support delete and we support Upsert -- so would be CRUU :)
2. May not be obvious to users